_posts/blog/2017-07-19-google-assistant-development-for-android-devs-part-2.md

```diff
@@ -59,9 +58,11 @@ Now the capabilities of the [official Node.js SDK](https://github.com/Ticketmast
         	ApiAiAction(req, resp).handleRequest(actionMap)
        }
     }
+```
 
 ### Actions in Java
 
+```java
```

Comment:
So this code block is still throwing an error in Liquid (Jekyll's templating engine) because of the double `{{` starting on line 90. The best way I know to avoid it is to wrap the whole code block in Liquid's "raw" tags:

    {% raw %}
      ```java
      // ...
      ```
    {% endraw %}

Ugly I know but without this it's omitting everything between the `{{` and `}}` in your code.